test/docs/coverage.patch

```diff
@@ -0,0 +1,31 @@
+--- src/CMakeLists.txt	2023-12-07 03:53:02.257526792 +0000
```

Comment:
do we need this file?

Reply:
This is for the coverage at the end, but I was looking at a way around this a few days ago - I have a few more things to do try and I'll let you know if we need this or not